IPython/core/display.py

```diff
@@ -475,8 +475,13 @@ class Latex(TextDisplayObject):
 
     def _repr_latex_(self):
         return self.data
+        
+        
+class MathML(TextDisplayObject):
```

Comment:
@minrk do you think it's better to have a separate `MathML` display class, or combine the functionality into the `Math` class with a switch to distinguish between formats?

We do the former for SVG, but the latter for JPG/PNG in the `Image` class. I could go either way on this.